MSAL/test/integration/native_auth/end_to_end/reset_password/MSALNativeAuthResetPasswordEndToEndTests.swift

```diff
@@ -25,21 +25,19 @@
 import Foundation
 import XCTest
 
-final class MSALNativeAuthResetPasswordEndToEndTests: MSALNativeAuthEndToEndBaseTestCase {
-
-    private let usernameOTP = ProcessInfo.processInfo.environment["existingOTPUserEmail"] ?? "<existingOTPUserEmail not set>"
-    
+final class MSALNativeAuthResetPasswordEndToEndTests: MSALNativeAuthEndToEndPasswordTestCase {
     // Hero Scenario 2.3.1. SSPR – without automatic sign in
```

Comment:
Hey @borgesmb, it seems that here it happens the same as with the SignUp E2E tests, the references to the Hero Scenarios are from the old Figma instead of the Acceptance criteria doc and the test plan. Can you update them to match the AC doc?

Reply:
Done.